photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -388,22 +390,11 @@ private Optional<EstimatedRobotPose> multiTagPNPStrategy(
             fieldToCamsAlt.add(tagPose.transformBy(target.getAlternateCameraToTarget().inverse()));
         }
 
-        Optional<Matrix<N3, N3>> cameraMatrixOpt = Optional.empty();
-        Optional<Matrix<N5, N1>> distCoeffsOpt = Optional.empty();
-
-        if (camera != null) {
+        if (camera != null && cameraMatrixOpt.isEmpty() && distCoeffsOpt.isEmpty()) {
```

Comment:
We should pass in the result of `camera.getCameraMatrix();`. A user passing in an empty camera mat/distortion mat should (IMO) mean uncalibrated, not "oh go try to fetch the default".

Reply:
Didn't even think about that, ill make sure to change it